internal/container/dockerfile.go

```diff
@@ -109,7 +143,11 @@ func findDockerfilesAt(dir string) []string {
 		return nil
 	})
 
-	fmt.Println("Found Dockerfiles: " + strings.Join(foundFiles, ", \n\t"))
+	if len(foundFiles) >= 1 {
+		//fmt.Println("Found Dockerfiles:\n\t" + strings.Join(foundFiles, ", \n\t"))
```

Comment:
not sure if that's commented by purpose or not needed any more

Reply:
re-enabled as per team decision.